pkgs/os-specific/serenity/common.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenvNoCC
+, buildPackages
+}:
+
+rec {
+  rev = "4d87072201efc4ef9086588255fbb1bb993872ab";
+
+  version = "unstable-2023-04-03";
+
+  # Fetch Serenity source and patch shebangs
+  src = buildPackages.stdenvNoCC.mkDerivation {
+    pname = "serenity-src";
+    inherit version;
+
+    src = buildPackages.fetchFromGitHub {
+      owner = "SerenityOS";
+      repo = "serenity";
+      sha256 = "Yb3j/eiGfARLFh1LFySnAFN/Z0xis6erphsLZLyUw9o=";
+      inherit rev;
+    };
+
+    installPhase = ''
+      cp -r . $out
+    '';
```

Comment:
Why not just attach the meta to the `fetchFromGitHub` directly?  This extra derivation doesn't seem to do anything.

Reply:
This derivation is entirely for the purposes of running `patchShebangs` on the shell scripts in the repo. I suppose this could be done with `fetchFromGitHub` and adding the hook manually?

Reply:
Ah, apologies, I missed the commment.